randovania/server/multiplayer/session_common.py

```diff
@@ -35,24 +33,23 @@ def emit_inventory_update(association: WorldUserAssociation):
         return
 
     flask_socketio.emit(signals.WORLD_BINARY_INVENTORY,
-                        (association.world.uuid, association.user.id, association.inventory),
+                        (str(association.world.uuid), association.user.id, association.inventory),
                         room=get_inventory_room_name(association),
                         namespace="/")
-    try:
-        inventory: RemoteInventory = construct_pack.decode(association.inventory, RemoteInventory)
-
-        flask_socketio.emit(
-            signals.WORLD_JSON_INVENTORY,
-            (association.world.uuid, association.user.id, {
-                name: {"amount": item.amount, "capacity": item.capacity}
-                for name, item in inventory.items()
-            }),
-            room=get_inventory_room_name(association),
-            namespace="/"
-        )
-    except construct.ConstructError as e:
-        logger().warning("Unable to encode inventory for world %s, user %d: %s",
-                         association.world.uuid, association.user.id, str(e))
+    # try:
+    #     inventory: RemoteInventory = construct_pack.decode(association.inventory, RemoteInventory)
+    #     flask_socketio.emit(
+    #         signals.WORLD_JSON_INVENTORY,
+    #         (association.world.uuid, association.user.id, {
+    #             name: {"amount": item.amount, "capacity": item.capacity}
+    #             for name, item in inventory.items()
+    #         }),
+    #         room=get_inventory_room_name(association),
+    #         namespace="/"
+    #     )
+    # except construct.ConstructError as e:
+    #     logger().warning("Unable to encode inventory for world %s, user %d: %s",
+    #                      association.world.uuid, association.user.id, str(e))
```

Comment:
why comment this out?

Reply:
I'd love to only emit this if there's someone listening, as it's not cheap for the server.
It's there as it's convenient for third party services to hook though.